src/topology/algebra/order/basic.lean

```diff
@@ -3,11 +3,9 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
```

Comment:
In the interest of a smaller diff on github (and more importantly, a better `blame`), can you make a PR that just renames `topology/algebra/order/basic` to `topology/order/basic` first?

Reply:
#17860